controllers/yawol-controller/loadbalancerset/loadbalancerset_controller_test.go

```diff
@@ -106,6 +106,30 @@ var _ = Describe("LoadBalancerSet controller", Serial, Ordered, func() {
 			machine := getChildMachines(ctx, &setStub)[0]
 			Eventually(isValidMachine(&setStub, &machine)).Should(BeTrue())
 		})
+		It("Should eventually have a master", func() {
```

Comment:
We should check the state before we set a machine as keepalived master and check also if the LastTransitionTime changes